### DISCUSSION TITLE
Tweaking syntax highlighting in terminal

### DISCUSSION THREAD

**Discussion by nyxonman** [kingToolbox/WindTerm#726] [Q&A] [answered]
Is it possible to customize the syntax highlighting on the terminal so that we can map certain texts to different colors
I know it can be done from session settings to choose linux, cmd, powershell... but can we tweak furthermore?

For example if i want NotUseful to appear in red color, how may i achieve this?

**Comment by eugenov** [comment, marked as answer]
Sure you can! Edit terminal/schemes/linux/linux.lexer file (or powershell/powershell.lexer, you got the point). Regex is not too hard to understand. But with every update this file will be ovewriten.

**Reply by nyxonman** [reply]
Thanks for the info. Now i can add my own substrings
